docs/configuration.md

```diff
@@ -327,8 +327,8 @@ _Default: Infinity_
 
 This parameter configures the maximum zoom level available. Set it to a lower value to prevent your users from zooming in too deeply.
 
-### removeOnResize
+### destroy
```

Comment:
I would move it to: https://github.com/marmelab/EventDrops/blob/master/README.md#interface